src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts

```diff
@@ -395,8 +389,26 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
     if (exportDetailsDto.timeDeliveryTo) {
       exportDetailsDto.timeDeliveryTo = this.parseStrToTime(exportDetailsDto.timeDeliveryTo, exportDate);
     }
+    if (exportDetailsDto.receivingStationId) {
+      exportDetailsDto.receivingStationId = this.getReceivingStationIdByName(exportDetailsDto.receivingStationId.toString());
+    }
+  }
+  public getReceivingStationIdByName(receivingStationName: string): number {
+    return this.exportInfo.allReceivingStations.find((element) => {
+      if (receivingStationName === element.name) {
+        return element;
+      }
+    }).id;
+  }
+  public getReceivingStationById(receivingStationId: number): string {
+    return receivingStationId
```

Comment:
I would simplify it
const name = this.exportInfo.allReceivingStations.find((element) => receivingStationId === element.id)?.name
return name || null;